ariadne/src/Ariadne/Wallet/Cardano/Kernel/DB/HdWallet.hs

```diff
@@ -122,19 +134,22 @@ deriveSafeCopySimple 1 'base ''HasSpendingPassword
 data HdRootId = HdRootId (InDb (Core.AddressHash Core.PublicKey))
   deriving (Eq, Ord)
 
+instance Prelude.Show HdRootId where
+  show (HdRootId (InDb (addrHash))) = show addrHash
```

Comment:
Hm, I am wondering if there are reasons why `Show` is not derived for `InDb` :thinking: